src/machines/java/com/enderio/machines/common/item/CapacitorBankItem.java

```diff
@@ -0,0 +1,111 @@
+package com.enderio.machines.common.item;
+
+import com.enderio.machines.common.block.CapacitorBankBlock;
+import net.minecraft.core.Direction;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.nbt.Tag;
+import net.minecraft.world.item.BlockItem;
+import net.minecraft.world.item.ItemStack;
+import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.common.capabilities.ForgeCapabilities;
+import net.minecraftforge.common.capabilities.ICapabilityProvider;
+import net.minecraftforge.common.util.LazyOptional;
+import net.minecraftforge.energy.IEnergyStorage;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Optional;
+
+public class CapacitorBankItem extends BlockItem {
+    public CapacitorBankItem(CapacitorBankBlock pBlock, Properties pProperties) {
+        super(pBlock, pProperties);
+    }
+
+    @Override
+    public @Nullable ICapabilityProvider initCapabilities(ItemStack stack, @Nullable CompoundTag nbt) {
+        return new BlockEntityEnergyStorage(stack, ((CapacitorBankBlock)getBlock()).getTier().getStorageCapacity());
+    }
+
+    private static class BlockEntityEnergyStorage implements IEnergyStorage, ICapabilityProvider {
```

Comment:
Why does this need a capability? If it is necessary then maybe we should be giving this to all of our powered blocks?

Reply:
I have made an itemstack energy capability, but that one does not store it in the block entity tag (we could maybe change that, but I'm not sure that is needed). The reason this one is different, is because our machines don't expose the energy when they are in item form, while the powerbanks due.

Reply:
Yeah I mean it'd be nice to break this out into its own class

Reply:
I would do that once we have the requirement, not now